.github/workflows/update-nix.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       - name: Update flake.lock
         uses: DeterminateSystems/update-flake-lock@v19
         with:
+          token: ${{ secrets.CREATE_PR_TOKEN }}
```

Comment:
I suppose this token is linked to your account right now. I would rather have it linked to some ONEKEY bot account.

Reply:
Yep, but at least it is organization managed, so can be revoked and checked from the org dashboard.

There is no "bot" account per se, it'd need to be a regular github account set-up with access to the organization.

Reply:
I mean an account like https://github.com/bedevere-bot

Reply:
https://github.com/onekey-sec/unblob/pull/612